uk.co.compucorp.civicrm.tasksassignments/tasksassignments.php

```diff
@@ -524,25 +525,41 @@ function _tasksAssignments_allow_only_add_timeline_action (phpQueryObject $doc)
  *
  * @param phpQueryObject $doc
  */
-function _tasksAssignments_remove_columns_from_timeline (phpQueryObject $doc) {
+function _tasksAssignments_remove_columns_from_timeline(phpQueryObject $doc) {
   $columnsToBeRemovedFromTimeline = [ 'Status', 'Reference', 'Select' ];
 
   foreach ($columnsToBeRemovedFromTimeline as $columnName) {
-    $columnHeader = $doc->find('table th:contains("' . $columnName . '"');
-    $columnIndex = $doc->find('table th')->index($columnHeader) + 1;
-    $columnBody = $doc->find('table td:nth-child('. $columnIndex .')');
+    _tasksAssignments_remove_column($doc, $columnName);
+  }
+}
+
+/**
+ * Removes a column from a table by using the column's header name as a reference.
+ *
+ * @param phpQueryObject $doc
+ * @param string $columnName
+ */
+function _tasksAssignments_remove_column(phpQueryObject $doc, $columnName) {
+  $columnHeader = $doc->find('table th:contains("' . $columnName . '")');
 
-    $columnHeader->remove();
-    $columnBody->remove();
+  // skip if the column does not exist:
+  if (!count($columnHeader)) {
```

Comment:
Seems `$columnHeader` is an object and not an array, count is usually associated with an array.
This should be enough:
```php
if(!$columnHeader) {
return;
}
```